### ISSUE TITLE
Broken quotation marks

### ISSUE THREAD

**Issue by PerWiklander** [UNECE/GSIM-2.0#9]
Compare Unit Type in the old spec format to the new. Where there were `"` or `'` before, we now have blank spaces.

Old:
![CleanShot 2024-03-20 at 10 49 13](https://github.com/UNECE/GSIM-2.0/assets/391989/7e9b3465-88dc-4775-983c-366043260d86)

New:
![CleanShot 2024-03-20 at 10 49 35](https://github.com/UNECE/GSIM-2.0/assets/391989/3679229f-8a3b-4c96-bd90-4abe2cc95fc0)

It looks like this has happened to all the text in the document.

Code view:
![CleanShot 2024-03-20 at 11 04 16](https://github.com/UNECE/GSIM-2.0/assets/391989/073469e3-6e9c-4b83-b236-7e35228eeb90)

An easy fix is to search and replace the `STS` and `CCH` characters with `&quot;` and `PU1` and `PU2` with `&apos;`, if nothing fancier is required.

**Comment by PerWiklander** [comment]
I just checked your link. On Safari/iPhone they are rendered as the infamous square box.

**Comment by PerWiklander** [comment]
![image](https://github.com/UNECE/GSIM-2.0/assets/391989/62400de0-da7d-4202-b1d1-a75f6c84f6bf)

**Comment by Chris-ECE** [comment]
Yes, I've since found Firefox renders them as blank spaces, though on Chrome and Edge they appear normally. - Let me investigate this further. Thanks

**Comment by Chris-ECE** [comment]
Okay, I'm thinking that `<q> and </q>` and `&lsquo; and &rsquo;` might be the way to go to preserve the curly quotes. - But I'm just a bit reluctant to lock us into a very specifically HTML-focused format for this content.

If no one minds, I think I'll put this issue on ice for a while.
